hydra.c

```diff
@@ -591,7 +591,8 @@ void help_bfg() {
          "             'A' for uppercase letters, '1' for numbers, and for all "
          "others,\n"
          "             just add their real representation.\n"
-         "  -y         disable the use of the above letters as placeholders\n\n"
+         "  -y         disable the use of the above letters as placeholders\n"
+         "  -r         use a method to delinearize the bruteforce.\n\n"
```

Comment:
that description basically tells the user nothing :) why should he/she use the option? and what would be the result?

Reply:
It's true that until you see an output it's hard to guess.
Maybe the idea of an increasing value used to modify the state and cut with a modulo, 
I'd have to explain that there is a slide in the sequence of increasing and decreasing the value.
The code is rain = i+3 in the loop on the length of the word,
When subtracting; for even words starting the loop at 1 and using i+1 can be explained as "subtract the same value that we added, minus (length-1) * 2; 
for odd words the loop for subtraction starts at 2 and it's end is inclusive, we are still using i+1 but i'm having a hard time seeing what it does.
I made a last fix that i'm pushing now/

Reply:
you have to change the description to something meaningful for the user. the text there now could also be "foo bar foo bar" :)